tests/errors/semantic/blocking/classes.py

```diff
@@ -0,0 +1,5 @@
+# pylint: disable=missing-function-docstring, missing-module-docstring, missing-class-docstring
+
+class MyClass:
+    def __init__(self : 'MyClass', param1 : 'int|float'):
+        print(param1)
```

Comment:
Please can you use names which explain what is being tested. E.g
```
git mv classes.py INTERFACE_CLASS__INIT__.py
```

Reply:
Fixed https://github.com/pyccel/pyccel/pull/1576/commits/70412aeb3da4844bce30dbdec815e675a5794da2.